suite-native/app/ios/TrezorSuite.xcodeproj/project.pbxproj

```diff
@@ -519,8 +519,10 @@
 			buildSettings = {
```

Comment:
Please, remove this file from PR. I think it was created during our experiment with a certificate for you. This is set up correctly in my PR https://github.com/trezor/trezor-suite/pull/5694/files#diff-2ef3b550847252fc53a8230cd697c2a211db2e204caf43c04f2bfd05ec3d2e08

Reply:
[b1317b5](https://github.com/trezor/trezor-suite/pull/5693/commits/b1317b5f68e058d0f402a83a4898ecb7f4655954)